wkcuber/api/Layer.py

```diff
@@ -151,6 +236,10 @@ def setup_mag(self, mag):
             self.name, mag, wk_header.block_len * wk_header.file_len
         )
 
+    def _initialize_mag_from_other_mag(self, new_mag_name, other_mag, compress):
+        block_type = wkw.Header.BLOCK_TYPE_LZ4HC if compress else wkw.Header.BLOCK_TYPE_RAW
+        return self.add_mag(new_mag_name, other_mag.block_len, other_mag.file_len, block_type)
```

Comment:
The new implementation uses the same `block_len` as the source magnification. In the previous implementation this always defaults to 32